rmw_fastrtps_shared_cpp/src/rmw_get_topic_endpoint_info.cpp

```diff
@@ -325,13 +121,28 @@ __rmw_get_subscriptions_info_by_topic(
   bool no_mangle,
   rmw_topic_endpoint_info_array_t * subscriptions_info)
 {
-  return _get_info_by_topic(
+  rmw_ret_t ret = _validate_params(
     identifier,
     node,
     allocator,
     topic_name,
-    no_mangle,
-    false, /*is_publisher*/
+    subscriptions_info);
+  if (ret != RMW_RET_OK) {
+    return ret;
+  }
+  auto common_context = static_cast<rmw_dds_common::Context *>(node->context->impl->common);
+  rmw_dds_common::GraphCache & graph_cache = common_context->graph_cache;
```

Comment:
@ivanpauno can be made `const`?

Reply:
See https://github.com/ros2/rmw_fastrtps/pull/312/commits/41977050b94d1e1498a35ba3f7c393048f0da8d5.